### PR TITLE
Add small fixture of development data

### PR DESCRIPTION
**Pull request by igarcez**
This MR adds an initial small data fixture that can be used to speed development or tests

### CODE REVIEW

File: fixtures/small.yaml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  fields:
+    deleted:
+    user: 5
+    type: AF
+    crisis: 1
+    first_line_of_address: Västerlånggatan 48
+    second_line_of_address: "-"
+    country: SE
+    place_id: 83FC+J6 Stockholm
+    position:
```
Comment (tonythomas01) [inline fixtures/small.yaml]: Oo, will this position go through ? I thought we had to have a geo point here ?

Comment (tonythomas01) [reply fixtures/small.yaml]: Without this, I get a feeling we might end up with an exception on calling v1/affected-participants/ ? Should we fix it ?

**Review by tonythomas01 — COMMENTED** [review]
Check comments, but great stuff.

File: fixtures/small.yaml (hunk 1 above)
Comment (tonythomas01) [reply fixtures/small.yaml]: I can see how a point field looks on the JSON though. https://stackoverflow.com/a/59539162 dont know how helpful that is.

Comment (igarcez) [reply fixtures/small.yaml]: Cool, I didn't know what to provied, what about the place_id, is that correct? I also didn't know what to provide

Comment (tonythomas01) [reply fixtures/small.yaml]: So you can see an example placeId on the API docs herehttp://docs.quarantinehelp.space/#/Authentication/post_api_v1_auth_register_   `"placeId": "ChIJwyyKo7J3X0YRZ5XOMcLx3xo"` It is more like a HERE maps placeId

Comment (igarcez) [reply fixtures/small.yaml]: @tonythomas01 pushed adjustments, can you please take a look again?

**Review by tonythomas01 — APPROVED** [review]
Good stuff. Lets see if someone has some issue loading this! Till then ✅ and tack tack.

**Pull request merged**